test/units/executor/test_playbook_executor.py

```diff
@@ -28,17 +25,17 @@
 from units.mock.loader import DictDataLoader
 
 
-class TestPlaybookExecutor(unittest.TestCase):
+class TestPlaybookExecutor():
```

Comment:
If this test is being converted, the unnecessary class should be removed as well.